src/website/src/releases/2.x/2.0.0.html

```diff
@@ -51,13 +51,13 @@ <h6 new-component>New Stepper Component (<a href="https://github.com/vmware/clar
             code in use check out the documentation <a routerLink="/documentation/stepper">here</a>.</p>
         <h6 new-component>Figma Design Resources For Clarity (beta)</h6>
         <p>Jumpstart your Clarity design with one of the Figma libraries available here:</p>
-        <a [href]="lightFigma" target="_blank"
+        <a href="https://www.figma.com/file/nRpy4E8bYy9hiDc8iORxyp/Clarity-UI-Library-light-2.0.0?node-id=262%3A7" target="_blank"
```

Comment:
As I mentioned in the slack channel. I would prefer to keep this bound to `lightFigma` and import the url string from `environment.prod.ts` into the `news.component.ts` file like it is in the `get-started.component.ts` file.

Reply:
But `environment.prod.ts` should only contain dev/prod environment-related data though. That's why I suggested a dedicated file for resource links which would contain this category of data.

Another problem is that `environment.prod.ts` is global so if we update the figma links properties in that file that will take effect in all versions of release notes unless we keep creating new properties for all versions of figma links in those files which we probably wouldn't want that.

Reply:
Another issue is that we should be showing the figma links in the release notes header box, sketch files are listed there now.

I'm ok with hard coding these examples since its part of the release notes and it would lock the version though.